docs/metrics/README.md

```diff
@@ -390,18 +392,19 @@ Given a metric, once the cardinality limit is reached, any new measurement which
 cannot be independently aggregated because of the limit will be dropped or
 aggregated using the [overflow
 attribute](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute)
-(if enabled). When NOT using the overflow attribute feature a warning is written
-to the [self-diagnostic log](../../src/OpenTelemetry/README.md#self-diagnostics)
-the first time an overflow is detected for a given metric.
+(if enabled). A warning is written to the [self-diagnostic
+log](../../src/OpenTelemetry/README.md#self-diagnostics) the first time an
+overflow is detected for a given metric.
 
 > [!NOTE]
 > Overflow attribute was introduced in OpenTelemetry .NET
-  [1.6.0-rc.1](../../src/OpenTelemetry/CHANGELOG.md#160-rc1). It is currently an
+  [1.6.0-rc.1](../../src/OpenTelemetry/CHANGELOG.md#160-rc1). It was an
```

Comment:
We don't need to maintain the experimental part in readme, this could be good to place in changelog.

Reply:
I've removed this section.